app/config/index.js

```diff
@@ -6,6 +6,8 @@ const schema = Joi.object({
   port: Joi.number().default(3000),
   env: Joi.string().valid('development', 'test', 'production').default('development'),
   staticCacheTimeoutMillis: Joi.number().default(15 * 60 * 1000),
+  restClientTimeoutMillis: Joi.number().default(20000),
```

Comment:
Two different ways of coming up with 20000. I have no preference as to which is used.

Reply:
one comes up with 900,000 (15 minutes) and the other is 20,000 (20 seconds). I'll update the README with the correct values and set to 20000 to 20 * 1000.